sections/featured-collection.liquid

```diff
@@ -68,24 +72,24 @@
           {%- endfor -%}
         {%- endfor -%}
       </ul>
-      {%- if show_mobile_slider -%}
-        <div class="slider-buttons no-js-hidden">
-          <button type="button" class="slider-button slider-button--prev" name="previous" aria-label="{{ 'general.slider.previous_slide' | t }}">{% render 'icon-caret' %}</button>
+      {%- if show_slider_on_mobile or show_slider_on_desktop -%}
+        <div class="slider-buttons no-js-hidden{% if show_slider_on_desktop == false %} medium-hide{% endif %}{% if show_slider_on_mobile == false %} small-hide{% endif %}">
```

Comment:
So, we're adding a `slider--desktop` class to the parent if the.

Can this logic be moved to CSS so we don't have to check it multiple times in Liquid?

This could be replaced with something like:
```css
@media screen and (min-width: 750px) and (max-width: 989px) {
  .slider:not(.slider--desktop) .slider-buttons { 
    display: none;
  }
}
```